### ISSUE TITLE
The application was unable to start correctly (0xc0000142). Click OK to close the application.

### ISSUE THREAD

**Issue by Slient1802** [deskflow/deskflow#7871]
### Project

Deskflow

### Deskflow version number

1.17.0+r1 (44bd69f)

### Build type

Deskflow package (downloaded from Deskflow)

### Operating systems (OS)

- [X] Windows
- [ ] macOS
- [ ] Linux (X11)
- [ ] Linux (Wayland)
- [ ] BSD-derived
- [ ] Other (please specify)

### Wayland on Linux

- [ ] I have reviewed the Wayland [known issues](https://github.com/deskflow/deskflow/discussions/7499) and my issue is new
- [X] I am not using Wayland on Linux

### Signing on macOS

- [ ] I have authorized the app to run on my Mac
- [X] I am not using macOS

### Continuous build

- [ ] I have tried the latest continuous build and the issue persists
- [x] I am unable to try the latest continuous build

### OS versions/distros

-Window 10  
-Window 11

### Deskflow configuration

Window 11 sever. Window 10 client (window 11 has 2 monitors and on the right of Window 10 - 1 monitor)

### What steps will reproduce the problem?

1. Start Deskflow
2. Link 2 computers together
3. Using deskflow to shut down my client computer

### Log output

```shell
[2024-11-08T12:20:15] WARNING: failed to determine current keyboard layout
[2024-11-08T12:20:15] DEBUG: open clipboard
[2024-11-08T12:20:15] DEBUG: close clipboard
[2024-11-08T12:20:15] DEBUG: ignored screen "RH-Lap" update of clipboard 0 (unchanged)
[2024-11-08T12:20:15] DEBUG: open clipboard
[2024-11-08T12:20:15] DEBUG: close clipboard
[2024-11-08T12:20:15] DEBUG: ignored screen "RH-Lap" update of clipboard 1 (unchanged)
[2024-11-08T12:20:15] DEBUG: dropped bogus delta motion: -1942,+517
[2024-11-08T12:20:16] INFO: switch from "Ruan-Slients-PC" to "RH-Lap" at 7,801
[2024-11-08T12:20:16] INFO: entering screen
[2024-11-08T12:20:17] INFO: switch from "RH-Lap" to "Ruan-Slients-PC" at 1919,854
[2024-11-08T12:20:17] INFO: leaving screen
[2024-11-08T12:20:17] WARNING: failed to determine current keyboard layout
[2024-11-08T12:20:17] DEBUG: open clipboard
[2024-11-08T12:20:17] DEBUG: close clipboard
[2024-11-08T12:20:17] DEBUG: ignored screen "RH-Lap" update of clipboard 0 (unchanged)
[2024-11-08T12:20:17] DEBUG: open clipboard
[2024-11-08T12:20:17] DEBUG: close clipboard
[2024-11-08T12:20:17] DEBUG: ignored screen "RH-Lap" update of clipboard 1 (unchanged)
[2024-11-08T12:20:17] DEBUG: dropped bogus delta motion: -1938,+623
[2024-11-08T12:20:18] INFO: switch from "Ruan-Slients-PC" to "RH-Lap" at 7,443
[2024-11-08T12:20:18] INFO: entering screen
[2024-11-08T12:20:23] NOTE: client "Ruan-Slients-PC" is dead
[2024-11-08T12:20:23] DEBUG: closing socket: BF6124C0
[2024-11-08T12:20:23] DEBUG: active sides: 2
```

### Additional information

The bug appears then I can't shutdown or doing anything by my sever mouse and keyboard to click OK button to fully shutdown until I use my client computer mouse or keyboard to press OK button 
Picture capture when the bug appears:
![z6011546533960_57ef39c0d49b7d38690cc92277f5b952](https://github.com/user-attachments/assets/c5c0a574-a36e-4e44-8bf1-5d733feee3b0)

**Comment by sithlord48** [comment]
Please try 1.17.1

**Comment by Slient1802** [comment]
> Please try 1.17.1

It seems that this problem has been solved in version 1.17.1.0 (a80cea3c)
